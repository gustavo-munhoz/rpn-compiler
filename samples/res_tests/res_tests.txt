(100 2 /)
(-8 2 *)
((1 RES) 10 +)
((3 RES) 20 -)
((2 RES) (4 RES) +)
(5 RES)