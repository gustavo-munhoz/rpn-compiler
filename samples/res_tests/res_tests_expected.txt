(100 2 /)                => 50 (0x5240)
(-8 2 *)                 => -16 (0xCC00)
((1 RES) 10 +)           => -6 (0xC600)
((3 RES) 20 -)           => 30 (0x4F80)
((2 RES) (4 RES) +)      => 44 (0x5180)
(5 RES)                  => 50 (0x5240)